README.adoc

```diff
@@ -311,6 +311,19 @@ The self-provision of an Elasticsearch cluster can be disabled by setting the fl
 
 IMPORTANT: At the moment there can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
 
+=== Elasticsearch index cleaner job
+
+When using `elasticsearch` storage by default a job is created to clean old traces from it, the options for it are listed below so you can configure it to your use case
+
```

Comment:
Could you add the code block demarcations, like 

```asciidoc
[source,yaml]
----
storage :
  type: ...
```

This way, we should get appropriate syntax highlighting in some of the supported outputs.

Reply:
Sure!